.gitignore

```diff
@@ -1,6 +1,7 @@
 ._*
 *.tgz
 *.phar
+.idea
```

Comment:
Please add this to your global .gitIgnore and remove it from the project, it's user specific.